test/test_core.py

```diff
@@ -1,7 +1,3 @@
-# Copyright 2013 The Emscripten Authors.  All rights reserved.
-# Emscripten is available under two separate licenses, the MIT license and the
-# University of Illinois/NCSA Open Source License.  Both these licenses can be
-# found in the LICENSE file.
```

Comment:
Not sure why this was deleted... revert?

Reply:
Hm yeah looks like unfortunate rebase artifact.